start-deploy.sh

```diff
@@ -186,23 +190,50 @@ if [[ "${DOWNSTREAM}" == "true" ]] || [[ "${INSTALL_ICSP}" == "true" ]]; then
   FULL_TOKEN="${DOWNSTREAM_QUAY_TOKEN}${OPENSHIFT_PULL_SECRET}"
   if [[ "${DOWNSTREAM}" == "true" ]]; then
     printlog info "Setting up for downstream deployment"
-    export COMPOSITE_BUNDLE=true
-    export CUSTOM_REGISTRY_REPO="quay.io:443/acm-d"
-    export QUAY_TOKEN=$(echo "${DOWNSTREAM_QUAY_TOKEN}" | ${BASE64})
+    export COMPOSITE_BUNDLE, CUSTOM_REGISTRY_REPO, QUAY_TOKEN
+    COMPOSITE_BUNDLE=true
+    CUSTOM_REGISTRY_REPO="quay.io:443/acm-d"
+    QUAY_TOKEN=$(echo "${DOWNSTREAM_QUAY_TOKEN}" | ${BASE64})
   else
     printlog info "Installing ICSP"
   fi
   printlog info "Updating Openshift pull-secret in namespace openshift-config with a token for quay.io:433"
   oc set data secret/pull-secret -n openshift-config --from-literal=.dockerconfigjson="$(jq -s '.[0] * .[1]' <<<"${FULL_TOKEN}")"
   printlog info "Applying downstream resources (including ImageContentSourcePolicy to point to downstream repo)"
   oc apply -k "${RHACM_DEPLOY_PATH}"/addons/downstream
+  printlog info "Applying ImageDigestMirrorSet"
+  oc apply -f - <<EOF
+apiVersion: config.openshift.io/v1
+kind: ImageDigestMirrorSet
```

Comment:
Will it now have both an ICSP and IDMS? I'd guess that will be fine, I just want to confirm that's intentional.

Reply:
It will, yes. I didn't really want to update the `deploy` repo or fiddle with something that's deprecated anyway, and they should play together just fine according to the docs starting in OCP v4.15.

ref: https://docs.redhat.com/en/documentation/openshift_container_platform/4.15/html/release_notes/ocp-4-15-release-notes#ocp-4-15-nodes-icsp-idms-compatibility